lib/linux.js

```diff
@@ -118,29 +119,15 @@ function _createScripts ( linux ) {
 }
 
 /**
- * Copy the app in the given folder
```

Comment:
The idea was to separate each steps of the building process, as it might involve more stuff over the time.